src/graphorm/index.ts

```diff
@@ -1497,6 +1496,19 @@ const obj = new GraphORM({
       },
     },
   },
+  PostAnalyticsPublic: {
+    from: 'PostAnalytics',
+    fields: {
+      impressions: {
+        rawSelect: true,
+        select: (_, alias) => {
+          return `
+            GREATEST(${alias}.impressions + ${alias}."impressionsAds", 0)
```

Comment:
```suggestion
            GREATEST(${alias}."impressions" + ${alias}."impressionsAds", 0)
```
just to be consistent

Reply:
ah missed the comment, its ok since lowercase but yes generally ok to put `"` everywhere